src/main/java/org/tc/models/usercourse/Attendee.java

```diff
@@ -0,0 +1,11 @@
+package org.tc.models.usercourse;
+
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "usercourse")
+@DiscriminatorValue("attendee")
+public class Attendee extends UserCourse {
```

Comment:
Attendee is a person. How he can be inherited from UserCourse? Looks like class name (Attendee) is not slightly correct.